chartfx-acc/src/main/java/de/gsi/acc/remote/util/Property.java

```diff
@@ -0,0 +1,161 @@
+package de.gsi.acc.remote.util;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Supplier;
+
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.gsi.acc.remote.RestServer;
+import de.gsi.dataset.remote.MimeType;
+
+import io.javalin.http.Handler;
+
+/**
+ * Property handler class -- PROTOTYPE/DRAFT for discussion/evaluation
+ * 
+ * intention: 
+ *  * wrapper around RESTful, CMW, other protocol backends to provide a consistent server-side API
+ * 
+ * @author rstein
+ *
+ * @param <G> GET/SUBSCRIBE data model -- a POJO definition
+ * @param <S> SET/NOTIFY data model -- a POJO definition
+ * @param <R> REQ/REP handler (contains connection details, back-end handler (default deprecated), method that are not needed for user-level server-side code, ...)
+ */
+public class Property<G, S, R> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Property.class);
+    private static final String ACCEPT_HEADER = "Accept";
+    private final String propertyName;
+    private final String htmlModelGet;
+    private final String htmlModelSet;
+    private final Handler handler;
```

Comment:
Codacy found an issue: [Perhaps 'handler' could be replaced by a local variable.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6305992)